flyway/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:18.04 as builder
```

Comment:
@tshinowpub 
The directory name should be aligned with `flyway-mysql56`.
If you choose to use the name `flyway`, make sure it is always the latest version by automatically updating it in `variantdev/mod`.

Reply:
https://github.com/chatwork/dockerfiles/pull/1011/commits/c523b239461e71d41931df71ff968c5dc0c91241